### PR TITLE
Support Laravel 10.x

### PR DESCRIPTION
**Pull request by mtawil**

### CODE REVIEW

**Comment by bradyrenting** [comment]
Thanks for your contribution @mtawil! 🙌

It seems like the Github workflow is currently failing, I will look into this 😅

**Comment by mtawil** [comment]
> Thanks for your contribution @mtawil! 🙌
> 
> It seems like the Github workflow is currently failing, I will look into this 😅

Hey, can you please try again?

**Comment by mtawil** [comment]
> Thanks for your contribution @mtawil! 🙌
> 
> It seems like the Github workflow is currently failing, I will look into this 😅

Hey @bradyrenting, can you please try again?

**Comment by mtawil** [comment]
@bradyrenting let me try to fix it 😅

**Comment by bradyrenting** [comment]
> @bradyrenting let me try to fix it 😅

Thanks haha. Let's just say that the workflow isn't the only one who's failing...

**Comment by mtawil** [comment]
@bradyrenting please approve the workflow to test

File: .github/workflows/run-tests.yml [hunk 1]
```diff
@@ -20,10 +20,10 @@ jobs:
           - laravel: 10.*
             testbench: 8.*
           - laravel: 9.*
-            testbench: 7.*
+            testbench: ^7.22
         exclude:
-          - laravel: 10.*
-            php: 8.0
+          - php: 8.0
```
Comment (bradyrenting) [inline .github/workflows/run-tests.yml:25]: Shouldn't this be like the following:
```exclude:
          - laravel: 10.*
            php: 8.0
```

Comment (mtawil) [reply .github/workflows/run-tests.yml:25]: I guess no because here we said to exclude laravel 10 on PHP 8.0

Comment (bradyrenting) [reply .github/workflows/run-tests.yml:25]: That's what it's supposed to do, right? 

Seems like this PR works, though: https://github.com/spatie/laravel-package-tools/pull/81

Comment (mtawil) [reply .github/workflows/run-tests.yml:25]: Oh okay so I believe it does the same thing, look here: [ryangjchandler/blade-capture-directive#12](https://github.com/ryangjchandler/blade-capture-directive/pull/12/files#diff-7314d0ebbd2e9537ae4889316745b4fd2fa43cb86275c9caae18a86ba228b642R24-R26)

**Comment by mtawil** [comment]
@bradyrenting approve plz

**Comment by bradyrenting** [comment]
That seems to be working! Thanks again 🙌

**Pull request merged**